packages/terra-table/src/SingleSelectableRows.jsx

```diff
@@ -47,17 +42,20 @@ class SingleSelectableRows extends React.Component {
 
   handleSelection(event, index) {
     this.setState({ selectedIndex: index });
+    if (this.props.onChange) {
+      this.props.onChange(event);
+    }
   }
 
   shouldHandleSelection(index) {
     return index !== this.state.selectedIndex;
   }
 
   wrappedOnClickForRow(row, index) {
-    const initialOnClick = this.props.onClick;
+    const initialOnClick = row.props.onClick;
```

Comment:
Should this still be onClick?

It looks like we are removing that prop here: https://github.com/cerner/terra-core/pull/392/files#diff-eccf8537e1685f6d5a1790de1307f59fL18